test/test-lib/fixtures.ts

```diff
@@ -14,7 +155,34 @@ export const setDefaultFixtures = (
 
 export type FixtureData = Dictionary<Dictionary<any>>;
```

Comment:
Nit, but might worth renaming this to `Fixtures`, since it's exported and is what tests will use, and the `Fixtures` registry above to `FixtureData` (or `FixturesRegistry`, or `FixturesByModel`, etc.).